README.md

```diff
@@ -57,5 +57,6 @@ CloudGraph Tencent Provider will ask you what regions you would like to crawl an
 | Service | Relations |
 | ------------------- | ------------------- |
 | securityGroup | | 
+| securityGroupRule | |
```

Comment:
Is a security group rule not associated with a sg?

Reply:
yeah some of these concepts are lost in translation, `securityGroup` is under `vpc` while `securityGroupRule` is under CFW (Cloud Firewall). So far tencent doesnt seem to care so much about connections between services since it hardly references other service IDs or any unifying account/project id in the schemas.